### DISCUSSION TITLE
Removing hard link without affecting original

### DISCUSSION THREAD

**Discussion by randalltrini** [ololx/quick-symlink#16] [General]
Hello,

I am a novice at this stuff. How can I remove a hardlink that I created without affecting the original? When I send to trash and attempt to delete, it appears to try delete the original folder...

**Comment by ololx** [comment]
Hello @randalltrini
It would be safe to assume that we are dealing with a misunderstanding. Just that I created and then deleted the hard link to the folder, and everything was as expected - the original folder was not deleted. 
Could you be so kind to add more details of your case?

**Reply by randalltrini** [reply]
I created a hard link using the quick-symlink app. I wanted to remove the hardlink so I sent it to the trash. When I tried to empty trash, it started to count number of files/folders to be deleted so I cancelled the delete request.

**Reply by ololx** [reply]
Alright, now it's clear.
1 - When you create a hard link to a folder, links to the files inside that folder are also created.
2 - In the file manager, a hard link appears as a regular file. It doesn't have any special markings or differences in the interface that would indicate that it is not an independent file, but a link.

Therefore, when deleting in the interface, you see the recalculation of the files being deleted (1)-(2), but these are specifically hard links, and deletion in macOS should not affect the original files.

**Reply by ololx** [reply]
I tried to recreate the situation with creating and deleting a link to a folder and show you the screenshots: 

1 - create hard link "workspace-1" for folder "workspace"
<img width="201" alt="Снимок экрана 2024-05-10 в 6 32 25 PM" src="https://github.com/ololx/quick-symlink/assets/57698089/4dfe87c3-7c0d-43ef-a870-466588ca70ed">

2 - move  hard link "workspace-1" to trash
<img width="643" alt="Снимок экрана 2024-05-10 в 6 32 54 PM" src="https://github.com/ololx/quick-symlink/assets/57698089/10985f94-402a-4a59-9411-ee21440f2f50">

3 - clear trash with counting files
<img width="444" alt="Снимок экрана 2024-05-10 в 6 33 00 PM" src="https://github.com/ololx/quick-symlink/assets/57698089/1d318cd4-028b-438a-81bc-c94c361895ee">

4 - original folder "workspace" is safe
<img width="371" alt="Снимок экрана 2024-05-10 в 6 33 15 PM" src="https://github.com/ololx/quick-symlink/assets/57698089/189b15b8-1018-4814-9c77-6037b9d264ed">

**Reply by randalltrini** [reply]
Thank you! Its is now clear in my mind I was able to successfully delete.

**Reply by ololx** [reply]
Glad to hear that. Have a good day!
